blocks/columns/columns.css

```diff
@@ -19,15 +15,93 @@
   display: block;
 }
 
+/*
+ * Column Manufacturing Steps
+*/
+
+.section.columns-container h2 {
+  font-size: 24px;
```

Comment:
I started this process on my PR - please use this approach. It's what we did on the [CommonMoves Project](https://github.com/hlxsites/hsf-commonmoves/blob/77127552b271de09db8364f4fb2dca04fa82a238/styles/styles.css#L58-L75).

Font sizes/Weights/Family/etc are variables, so we can change meaning one place.